ompi/mca/pml/ucx/pml_ucx.c

```diff
@@ -1006,3 +1016,8 @@ int mca_pml_ucx_dump(struct ompi_communicator_t* comm, int verbose)
 {
     return OMPI_SUCCESS;
 }
+
+static void mca_pml_ucx_mpi_init_bottom(int argc, char **argv, int requested, int *provided)
+{
+    *provided = MPI_THREAD_SINGLE;
```

Comment:
do we assume that if UCX was not unloaded, it means it was selected as PML?

Reply:
right, OMPI unloads all non-selected PML's

Reply:
there is cleanup loop:
https://github.com/open-mpi/ompi/blob/ae2bda9aa045c4dca7f1cbd12f2a20a084336b60/ompi/mca/pml/base/pml_base_select.c#L206

Reply:
i think the fact that unused components are unloaded is more of an optimization, and we better not rely on that (is this still the case in static compilation?)
need a better way to know if ucx was selected
Maybe we can set thread support level when pml_ucx module is initialized?

Reply:
hmm, there is no other way to know is component selected or not (except component is closed), list of opened components is local in ```select``` function and we may be sure that unselected components are cleaned

Reply:
can we check if a ucx module was created?

Reply:
@yosefe done